### PR TITLE
Derive Serialize for Packages, Vulnerabilities, and Vulnerability

### PR DESCRIPTION
**Pull request by colindean**
This will support the output of JSON from cargo-audit.

Work toward RustSec/cargo-audit#40

### CODE REVIEW

**Comment by colindean** [comment]
This and a release of 0.10 is blocking https://github.com/RustSec/cargo-audit/pull/41.

File: src/vulnerability.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ use lockfile::Lockfile;
 use package::Package;
 
 /// A collection of vulnerabilities (i.e. matching advisories) for a given `Lockfile`
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
```
Comment (tarcieri) [inline src/vulnerability.rs]: Should probably derive `Deserialize` here too (and ditto for `Vulnerability`)

Comment (colindean) [reply src/vulnerability.rs]: ✅

**Comment by colindean** [comment]
I added the Deserialize.

**Comment by colindean** [comment]
Is anything else needed for this?

**Comment by tarcieri** [comment]
I'm currently visiting friends/family. Can take a look at these soon

**Comment by colindean** [comment]
No worries, no rush. Enjoy!

**Comment by colindean** [comment]
Ping!

**Comment by tarcieri** [comment]
Heading back from RustConf. Can take a look soon

**Comment by tarcieri** [comment]
Sorry about the delay on this. Will get another release of `rustsec` out soon, but I want to address https://github.com/RustSec/cargo-audit/issues/32 first

**Pull request merged**